charts/kubescape-operator/values.yaml

```diff
@@ -451,14 +453,18 @@ nodeAgent:
   image:
     # -- source code: https://github.com/kubescape/node-agent
     repository: quay.io/kubescape/node-agent
-    tag: v0.2.15
+    tag: v0.2.17
     pullPolicy: IfNotPresent
 
   config:
-    maxLearningPeriod: 3h # duration string
+    maxLearningPeriod: 12h # duration string
     learningPeriod: 2m # duration string
     updatePeriod: 10m # duration string
+    prometheusExporter: disable
+    httpExporterConfig: 
+      url: http://synchronizer.kubescape.svc.cluster.local:8089/apis/v1/kubescape.io/v1/runtimealerts
```

Comment:
what if the namespace is not `kubescape`?

Reply:
It's a good point.
We will change this in the next release